src/components/OrganizationDetailPanel.tsx

```diff
@@ -29,6 +38,15 @@ const OrganizationDetailPanel = ({
 					</PanelTitleTags>
 				</PanelTitleWrapper>
 			</PanelHeader>
+			<PanelGridItem>
```

Comment:
My `Panel`-related ergonomics aren't well-documented or storied, so this isn't your fault, but this should be a `PanelBody` not a `PanelGridItem`.

`PanelGrid` and `PanelGridItem` are both "invisible" wrapping elements that define the layout relationship among panels. You would never use them _inside_ a `Panel`.

Here's an example structure for a page with two panels: sidebar and main. Note that while some of these are optional — you don't need a `PanelHeader`, or you could make up a new element instead of using `PanelTitle` or `PanelActions` — the basic `PanelGrid` → `PanelGridItem[]` → `Panel` hierarchy is rigid.

```jsx
<PanelGrid sidebarred>
	<PanelGridItem>
		<Panel>
			<PanelBody>
				Sidebar contents
			</PanelBody>
		</Panel>
	</PanelGridItem>
	<PanelGridItem>
		<Panel>
			<PanelHeader>
				<PanelTitle>
					Main panel title
				</PanelTitle>
				<PanelActions>
					{/* Panel-related actions go here */}
				</PanelActions>
			</PanelHeader>
			<PanelBody>
				Main panel contents
			</PanelBody>
		</Panel>
	</PanelGridItem>
</PanelGrid>
```